src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -330,10 +328,9 @@ describe('AwsSamDebugConfigurationProvider', async () => {
                     runtime: validRuntime,
                 },
             }
-            ;(c as any).__noInvoke = true
             const actual = (await debugConfigProvider.resolveDebugConfiguration(folder, c))!
             const expected: SamLaunchRequestArgs = {
-                type: 'node', // Input "aws-sam", output "node".
+                type: AWS_SAM_DEBUG_TYPE,
```

Comment:
`type` is now set to "node"/"python"/"coreclr" in `SamDebugSession.attachRequest()`. This unfortunately can't be done before then (in the CONFIG phase) because otherwise our DebugAdapter would not get routed-to by VSCode.

Reply:
Seems kind of weird to mutate the field, should we just have a separate `runtimeFamily` field, or is it necessary to mutate this?

Reply:
vscode uses this field to choose a DebugAdapter